src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -13,45 +12,41 @@
   private Encoder rightEncoder;
   private Gearbox leftGearbox;
   private Gearbox rightGearbox;
-  private SpeedControllerGroup leftGearboxController;
-  private SpeedControllerGroup rightGearboxController;
-
   private Gyroscope gyroscope;
   private DifferentialDrive differentialDrive;
 
   //variables
   private double desiredAngle;
 
   //constants
-  private final double P_VALUE = .0025;//.0012
+  private final double P_VALUE = .0025;
+  private final double RAMP_RATE = 0.5;
+  private final double QUICK_TURN_THROTTLE_DEADZONE = 0.2;
   // left gear box CAN ids
   private final int LEFT_BACK_CAN_ID =11; 
-  // private final int LEFT_MIDDLE_CAN_ID = 13;
```

Comment:
We need the middle can IDs because our robot will have 3 motors

Reply:
ok, nbd, can add it when testing tonight.